### PR TITLE
tests: fix more tests for using ua-apps token

### PR DESCRIPTION
**Pull request by orndorffgrant**
Lets not merge this until we've finished running all tests against staging and are sure there aren't any more similar bugs

To test:
Please run the edited integration tests

After merging, I will cherry-pick to `release-27` and push directly to github without a PR if thats okay

### CODE REVIEW

**Comment by orndorffgrant** [comment]
Look at me with typos in my commit messages, fixing...

File: features/attached_enable.feature [hunk 1]
```diff
@@ -273,6 +273,7 @@ Feature: Enable command behaviour when attached to an UA subscription
         Then stdout matches regexp:
         """
         cis          +yes      +disabled +Center for Internet Security Audit Tools
+        esm-apps     +yes      +enabled  +UA Apps: Extended Security Maintenance \(ESM\)
```
Comment (renanrodrigo) [inline features/attached_enable.feature:276]: Is it expected that esm-apps will be on this list, even without the `--all`, being a beta service?

Comment (orndorffgrant) [reply features/attached_enable.feature:276]: Yep, When a beta service is enabled, then it will show up in this list with or without `--all`

Comment (renanrodrigo) [reply features/attached_enable.feature:276]: Ack!

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**